.github/CODEOWNERS

```diff
@@ -6,28 +6,34 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/ 
```

Comment:
I guess I own this. It was something hastily threw together by Trevor and I to work around TLS issues during verification on Mainnet